src/main/java/cms/gov/madie/measure/resources/AdminController.java

```diff
@@ -171,6 +178,128 @@ public ResponseEntity<List<Map<String, Object>>> getMeasureSharedWith(
     return ResponseEntity.ok(results);
   }
 
+  @PutMapping("/measures/{id}")
+  @PreAuthorize("#request.getHeader('api-key') == #apiKey")
+  public ResponseEntity<Measure> correctMeasureVersion(
+      HttpServletRequest request,
+      @Value("${admin-api-key}") String apiKey,
+      Principal principal,
+      @PathVariable String id,
+      @RequestParam String inCorrectVersion,
+      @RequestParam String correctVersion,
+      @RequestParam String draftVersion) {
+
+    // can version number be anything : eg: 430.0.001 or does it have to follow rules (like it
+    // should be draft of measureSetId)
+    // clear: for qdm: cqmMeasure, export, for qiCore: export
```

Comment:
were these the questions for POs?

Reply:
Oh my bad these were the question for POs, let me remove them